### DISCUSSION TITLE
配置排错

### DISCUSSION THREAD

**Discussion by Theddeus** [XTLS/Xray-core#472] [Q&A]
请问服务器log显示“tls错误，证书为空”是否可能因为使用openssl的自签名证书导致。谢谢

**Comment by MuenYu** [comment]
xray 允许使用 自签名证书，请提供更详细的信息以便进一步排错。

**Reply by Theddeus** [reply]
谢谢关注！经过排错,错因是赋权命令“+”错写为“-”导致实际未给证书赋予可读权限。
